templates/commands/goldentest/new_test_flags.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package goldentest
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/abcxyz/abc/templates/common/flags"
+	"github.com/abcxyz/pkg/cli"
+)
+
+// NewTestFlags describes what new golden test to render and how.
+type NewTestFlags struct {
+	// Positional arguments:
+
+	// Location is the file system location of the template to be tested.
+	//
+	// Example: t/rest_server.
+	Location string
+
+	// NewTestName are the name of the test case to create.
+	NewTestName string
```

Comment:
I think we should enforce that this name doesn't contain any slashes or backslashes. Those will cause problems with filesystem operations.

Reply:
done